cookie.go

```diff
@@ -25,6 +25,7 @@ func (c *Cookie) Set(ctx *fasthttp.RequestCtx, name string, value []byte, domain
 	cookie.SetHTTPOnly(true)
 	cookie.SetDomain(domain)
 	cookie.SetValueBytes(value)
+	cookie.SetSecure(secure)
```

Comment:
Only set a secure cookie if `shouldBeSecure` is true

Reply:
yes, fine.